projects/vdk-plugins/vdk-server/src/vdk/plugin/server/server_plugin.py

```diff
@@ -56,26 +57,29 @@
     help="Returns whether a local Control Service is currently installed.",
 )
 def server(install, uninstall, status):
-    flags = 0
-    if install:
-        flags += 1
-    if uninstall:
-        flags += 1
-    if status:
-        flags += 1
-
-    if flags != 1:
-        click.echo(
-            "Exactly one of --install, --uninstall, or --status options should be specified"
-        )
-    else:
-        installer = Installer()
+    try:
+        flags = 0
         if install:
-            installer.install()
-        elif uninstall:
-            installer.uninstall()
+            flags += 1
+        if uninstall:
+            flags += 1
+        if status:
+            flags += 1
+        if flags != 1:
+            click.echo(
+                "Exactly one of --install, --uninstall, or --status options should be specified"
+            )
         else:
-            installer.check_status()
+            raise Exception("paul test")
+            installer = Installer()
+            if install:
+                installer.install()
+            elif uninstall:
+                installer.uninstall()
+            else:
+                installer.check_status()
+    except Exception as e:
+        log.exception("VDK CLI command failed")
```

Comment:
Why don't we print the caught exception as well

Reply:
The exception is printed when you all this function. Some magic stuff.